extensions/blocks/rating-star/editor.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import { registerBlockType } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+import { StarIcon, StarBlockIcon } from './icon';
+import './editor.scss';
+import './style.scss';
+
+registerBlockType( 'jetpack/rating-star', {
```

Comment:
You should use `registerJetpackBlock` to ensure this block is using Jetpack's block availability framework. See example:

https://github.com/Automattic/jetpack/blob/54d177da89d95b24f77f8886bf8061ad7cb51661/extensions/blocks/business-hours/editor.js#L4-L7

Note also how other blocks are structured as `index.js` and `editor.js`

Reply:
done at https://github.com/Automattic/jetpack/pull/14091/commits/afc501de25bf66c5b29db80859026cd6e3cf56eb